src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/categories-property/categories-property.component.ts

```diff
@@ -34,6 +36,8 @@ export class CategoriesPropertyComponent extends BaseComponent implements OnInit
             ? this.dotMessageService.get('contenttypes.field.properties.category.label')
             : this.property.value;
             this.paginationService.url = 'v1/categories';
+        this.groupLocalCopy = _.cloneDeep(this.group);
```

Comment:
Why? what's the problem here?

Reply:
why do we need this copy?

Reply:
at this moment `categories` comes with a categories object (categoryName, description, inode, key, keywords, sortOrder) and to save it correctly we need on that property only the `inode`.

But we also need to pass that same categories object to the `dot-searchable-dropdown`, so I made a localcopy, to pass it to the dropdown and then update the parent categories attribute with the inode, so once it gets saved the endpoint doesn't break